### PR TITLE
disable test_poll2 and test_pidfd_open

### PR DESCRIPTION
**Pull request by salsal97**
Temporary fix for issues https://github.com/deislabs/mystikos/issues/1480 and https://github.com/deislabs/mystikos/issues/1481
Validation run (since nightly): https://openenclaveci.westus.cloudapp.azure.com/blue/organizations/jenkins/Mystikos%2FStandalone-Pipelines%2FSolutions-Tests-Pipeline/detail/Solutions-Tests-Pipeline/3006/pipeline/

### CODE REVIEW

**Review by asvrada — APPROVED** [review]

File: solutions/cpython-tests/readme.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ Here is a list of all the skipped individual unit tests
   * test_mkfifo_dir_fd
   * test_sched_rr_get_interval
   * test_sched_priority
+* test_poll
```
Comment (CyanDevs) [inline solutions/cpython-tests/readme.md:59]: Nit: align spacing

**Review by CyanDevs — APPROVED** [review]

File: solutions/cpython-tests/readme.md (hunk 1 above)
Comment (salsal97) [reply solutions/cpython-tests/readme.md:59]: Thats on purpose, its a separate category

File: solutions/cpython-tests/test_config_v3.8.11/patch [hunk 2]
```diff
@@ -84,6 +84,18 @@ index effac97..7d8cb89 100644
...
          # unlock it and exit.
+diff --git a/Lib/test/test_poll.py b/Lib/test/test_poll.py
+index ef966bf..34a227b 100644
+--- a/Lib/test/test_poll.py
++++ b/Lib/test/test_poll.py
+@@ -117,6 +117,7 @@ class PollTests(unittest.TestCase):
+     # Another test case for poll().  This is copied from the test case for
+     # select(), modified to use poll() instead.
+ 
++    @unittest.skip("Temporarily disabled in Mystikos")
```
Comment (vtikoo) [inline solutions/cpython-tests/test_config_v3.8.11/patch:95]: Could you add the reason for disabling - unsupported mmap from child before execve

File: solutions/cpython-tests/test_config_v3.9.7/patch [hunk 3]
```diff
@@ -166,6 +166,14 @@ index f4edb8b..924582c 100644
      def test_sched_rr_get_interval(self):
          try:
              interval = posix.sched_rr_get_interval(0)
+@@ -1469,6 +1480,7 @@ class PosixTester(unittest.TestCase):
+         open(fn, 'wb').close()
+         self.assertRaises(ValueError, os.stat, fn_with_NUL)
+ 
++    @unittest.skip("Temporarily disabled in Mystikos")
```
Comment (vtikoo) [inline solutions/cpython-tests/test_config_v3.9.7/patch:173]: Add reason - SYS_pidfd_open not supported in Mystikos.
Also do we need to skip for cpython 3.8?

Comment (salsal97) [reply solutions/cpython-tests/test_config_v3.9.7/patch:173]: It fails only in the 3.9 case

**Review by radhikaj — APPROVED** [review]

File: solutions/cpython-tests/readme.md (hunk 1 above)
Comment (CyanDevs) [reply solutions/cpython-tests/readme.md:59]: Ah, gotcha. My bad for viewing this in raw format!

**Pull request merged**